circle.yml

```diff
@@ -21,8 +21,10 @@ deployment:
     production:
         tag: /v[0-9]+(\.[0-9]+)*/
         commands:
-            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/docs/ --delete
+            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/$CIRCLE_TAG/docs/ --delete
+            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/latest/docs/ --delete
     develop:
         branch: master
         commands:
-            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/dev/docs/ --delete
+            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/dev/$CIRCLE_TAG/docs/ --delete
```

Comment:
Looking through the Circle env var [docs](https://circleci.com/docs/1.0/environment-variables/#build-details) it looks like the `$CIRCLE_TAG` var may only be defined for circle builds launched by a git tag push (I'm not certain about this). If it is the case, then it won't work here. 

I wonder if it would be better to use `$CIRCLE_SHA1` for development docs instead? Then we'd keep a record of each commit on dev, not just the latest. @seanrankine thoughts?